### PR TITLE
y importing multiple times XD check that time too 

### PR DESCRIPTION
**Pull request by niteshraj2310**
Signed-off-by: NiteshXD <niteshraj231@outlook.com>

### CODE REVIEW

**Comment by abhinav6497** [comment]
@niteshraj2310 Chemma is lub😘

File: userbot/modules/scrapers.py [hunk 1]
```diff
@@ -27,7 +26,6 @@
 import barcode
 from barcode.writer import ImageWriter
 import emoji
-from googletrans import Translator
```
Comment (sahyam2019) [inline userbot/modules/scrapers.py:30]: Yeh line kidhar hai?iske kyu hataya

Comment (niteshraj2310) [reply userbot/modules/scrapers.py:30]: it's there sir see karefully

Comment (sahyam2019) [reply userbot/modules/scrapers.py:30]: Line?

Comment (niteshraj2310) [reply userbot/modules/scrapers.py:30]: line 30  and  48 check

**Review by sahyam2019 — COMMENTED** [review]
Okay

**Pull request merged**